src/Command/Branch/BranchForkCommand.php

```diff
@@ -49,20 +56,15 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $adapter = $this->getAdapter();
-        $orgFork = $input->getArgument('other_organization');
-        $username = $this->getParameter('authentication')['username'];
-
-        $fork = $adapter->createFork($orgFork);
-
-        $repo = $input->getOption('repo');
         $sourceOrg = $input->getOption('org');
-        $targetOrg = $orgFork ?: $username;
+        $repo = $input->getOption('repo');
+        $targetOrg = $input->getArgument('target_organization');
```

Comment:
does this have a $input->getArgument('target_organization', defaultValueHere); thing?